css/src/components/accordion.scss

```diff
@@ -0,0 +1,156 @@
+/* stylelint-disable selector-no-qualifying-type */
```

Comment:
Regarding the sizing: after reviewing fluent's figma and sites, it turned out the only thing changes is the font size of the summary, and nothing else. And the content's size stays the same for all variations and is 14px (need to be updated here). So I'd say keep it simple and follow the same pattern

Reply:
✅ Updated.